rocm_bandwidth_test.cpp

```diff
@@ -587,12 +589,21 @@ void RocmBandwidthTest::RunCopyBenchmark(async_trans_t& trans) {
         hsa_signal_store_relaxed(signal_start_bidir, 1);
       }
 
-      // Create a timer object and reset signals
+      // Temporary code for testing
+      if (sleep_time_ > 0) {
+        std::this_thread::sleep_for(sleep_usecs_);
+      }
+
+      // Create a timer object and start it
       PerfTimer timer;
-      uint32_t index = timer.CreateTimer();
+      uint32_t cpuTimerIdx = 0;
+      if (print_cpu_time_) {
+        timer.InitTimer();

```

Comment:
FWIW - Please take a look at https://github.com/RadeonOpenCompute/rocm_bandwidth_test/pull/60 on using how to compute TSCFreq just once instead of every time inside a huge triple-nested loop.
Code review could catch such issues.

Reply:
This changelist is the first part. The plan is to completely remove calls to RDTSCP and instead use Chrono library service from C++